### PR TITLE
fix: [PIE-3144]: Handle longer policy set name

### PR DESCRIPTION
**Pull request by depanshu-dhiman**
##### Summary:

If only single policy added in a set, then try to show it fully without truncating by using the total width of 2 policies.
If more than 2 policies, then cap each policy width at 100px.
Adjust line height because of smaller font size and line clamp issue.
Use proper margin for numeric box.

##### Jira Links:

https://harness.atlassian.net/browse/PIE-3144

##### Screenshots:

Screenshot 2022-03-24 at 7 42 43 PM
Screenshot 2022-03-25 at 3 12 09 PM
Screenshot 2022-03-25 at 3 12 42 PM
Screenshot 2022-03-25 at 3 12 55 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx [hunk 1]
```diff
@@ -104,26 +104,36 @@ export function MiniPoliciesRenderer({ policies }: MiniPoliciesRendererProps) {
   const policyNames = policies.map(policy => policy.name)
 
   return (
-    <Layout.Horizontal flex={{ justifyContent: 'flex-start', alignItems: 'center' }} margin={{ right: 'small' }}>
-      {policyNames.slice(0, 2).map((policy, index) => (
-        <Text className={css.styledPolicy} key={index} lineClamp={1}>
+    <Layout.Horizontal flex={{ justifyContent: 'flex-end', alignItems: 'center' }} margin={{ right: 'small' }}>
+      {policyNames.slice(0, 2).map(policy => (
+        <Text
+          className={css.styledPolicy}
```
Comment (devinder-harness) [inline src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx:110]: `FontVariation` is missing

File: src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx [hunk 2]
```diff
@@ -104,26 +104,36 @@ export function MiniPoliciesRenderer({ policies }: MiniPoliciesRendererProps) {
...
+          font={{ size: 'small' }}
+          padding={{ top: 'xsmall', bottom: 'xsmall', left: 'small', right: 'small' }}
+          key={policy}
+          lineClamp={1}
+          style={{ maxWidth: length === 1 ? '240px' : '100px', lineHeight: '1.5em' }}
```
Comment (devinder-harness) [inline src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx:115]: can we use `classnames`  here?

**Review by devinder-harness — CHANGES_REQUESTED** [review]

File: src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx (hunk 2 above)
Comment (depanshu-dhiman) [reply src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx]: done

File: src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx (hunk 1 above)
Comment (depanshu-dhiman) [reply src/modules/75-cd/components/PipelineSteps/PolicyStep/PolicySets/PolicySetListRenderer/MiniPolicySetRenderer.tsx]: done, used font size variable in css

**Review by AshwinBhatkal-Harness — DISMISSED** [review]

**Review by PratyushGar — APPROVED** [review]

**Review by devinder-harness — DISMISSED** [review]

**Review by devinder-harness — APPROVED** [review]

**Review by AshwinBhatkal — APPROVED** [review]

**Review by AshwinBhatkal-Harness — APPROVED** [review]

**Pull request merged**